dashboard/test/models/blockly_test.rb

```diff
@@ -209,6 +209,55 @@ def assert_equal_xml(a, b)
     assert_equal localized_custom_block, translated_block
   end
 
+  test 'return default value if string is not translated' do
+    test_locale = :"te-ST"
+    I18n.locale = test_locale
+    custom_i18n = {
+      "data" => {
+        "blocks" => {
+          "DanceLab_jumpTo" => {
+            "text" => "springen naar {TIMESTAMP} {DIRECTION}",
+            "options" => {
+              "DIRECTION" => {
+                "right": "",
+                "left": "",
+              }
+            }
+          }
+        }
+      }
+    }
+
+    I18n.backend.store_translations test_locale, custom_i18n
+
+    level = create(:level, :blockly, level_num: 'level1_2_3')
+
+    custom_block =
+      [{
+        name: "DanceLab_jumpTo",
+        pool: "SelectDirection",
+        category: "Events",
+        config:
+          {
+            "color" => [140, 1, 0.74],
+            "func" => "atDirection",
+            "blockText" => "jump to {TIMESTAMP} {DIRECTION}",
+            "args" => [
+              {"name" => "TIMESTAMP", "type" => "Number", "field" => true},
+              {"name" => "DIRECTION", "options" => [["right", "right"], ["left", "left"]]}
```

Comment:
nit; when this test eventually catches a regression, it'll be a lot more clear what's wrong if the keys and values are distinct